www/src/routes/ResumeReview.tsx

```diff
@@ -1,8 +1,31 @@
+import React, { FC, useEffect } from 'react';
+import axios from 'axios';
 import { useAuth0 } from '@auth0/auth0-react';
-import React, { FC } from 'react';
+import config from '../util/config';
 
 const ResumeReview: FC = () => {
-    const { isAuthenticated } = useAuth0();
+    const { isAuthenticated, getAccessTokenSilently } = useAuth0();
+
+    useEffect(() => {
+        (async () => {
+            try {
+                const token = await getAccessTokenSilently({
+                    audience: config.server.audience,
+                    scope: 'call:ping',
```

Comment:
What happens if the user doesn't have the `call:ping` scope? Will it just get an access token but get `403`'s at the server. OR will it fail to get access token from Auth0?

Reply:
One of two things can happen. 

* If the user is assigned a role with the specified permissions but doesn't have it in the initial token, the error will say "Consent required"
* If the user doesn't have permissions, it'll say insufficient scope

Reply:
I'm hoping it throws a differrent exception so that we can display a different message for each

Reply:
Okay, and so it seems that this will all happen client side i.e. you don't need to hit the api with invalid token to learn that it is invalid.